src/app/services/firebase.service.ts

```diff
@@ -118,9 +118,8 @@ export class FirebaseService {
    * @param sessionID the ID of the session the students are in
    */
   getStudentsInSession(sessionID: string): Observable<SessionStudentData[]> {
-    return this.angularFirestore
-      .collection('sessions')
-      .doc(sessionID).collection<SessionStudentData>('students')
+    return this.angularFirestore.collection('sessions').doc(sessionID)
+      .collection<SessionStudentData>('students')
       .valueChanges({idField: 'id'});
```

Comment:
Do we need to be adding the id field at this point? I don't see it being used anywhere.

Reply:
If I remember correctly we were using it when assigning bees at the start of a round